classes/class-gf-securesubmit.php

```diff
@@ -2456,7 +2456,9 @@ private function create_customer($feed, $submission_data, $entry)
         $this->log_debug(__METHOD__ . '(): Customer meta to be created => ' . print_r($acctHolder, 1));
 
         /** @var string $modifier This value helps semi uniqely identify the customer */
-        $modifier = $this->getSecureSubmitJsResponse()->last_four . $this->getSecureSubmitJsResponse()->card_type;
+        $temp_card_last_4 = isset($this->getSecureSubmitJsResponse()->details->cardLast4) ? $this->getSecureSubmitJsResponse()->details->cardLast4 : $this->getSecureSubmitJsResponse()->last_four;
+        $temp_card_type = isset($this->getSecureSubmitJsResponse()->details->cardType) ? $this->getSecureSubmitJsResponse()->details->cardType : $this->getSecureSubmitJsResponse()->card_type;
```

Comment:
For improved readability, can we break these two lines up? Maybe something like:

```
`$temp_card_last_4 = isset($this->getSecureSubmitJsResponse()->details->cardLast4) ?
            $this->getSecureSubmitJsResponse()->details->cardLast4 :
            $this->getSecureSubmitJsResponse()->last_four;`
```